fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java

```diff
@@ -268,9 +270,14 @@ protected static String getLocation(final HttpResponse response) {
     }
 
     protected String getContentType(final HttpUriRequest method) throws ClientProtocolException, IOException {
```

Comment:
This exception can be removed from the `throws` clause: 'ClientProtocolException'